tests/conftest.py

```diff
@@ -165,11 +167,16 @@ def tear_down_hermitian():
 
 
 @pytest.fixture
-def in_tape_mode():
+def non_tape_mode_only():
```

Comment:
Why are we keeping that option open? :) To avoid deleting tests for now?

Reply:
Yep 😆